modules/tableEmbed.ts

```diff
@@ -12,11 +12,17 @@ export type TableRowColumnOp = Omit<Op, 'insert'> & {
 };
 
 export interface TableData {
-  rows?: Delta['ops'];
-  columns?: Delta['ops'];
+  rows?: Delta;
```

Comment:
We should not change the declaration of a public module. Here `Delta` should be a `Delta['ops']`.

Reply:
The problem here is that `tableEmbed.ts` is internally inconsistent. Functions like `compactTableData()` take in a `TableData` type but the _code_ expects `rows` and `columns` here to be Deltas (it calls `rows.length()` instead of checking `rows.length`, as we would expect if the type _really_ was an array of ops). So we have two options here:
1. Change the declaration of the public module to match what the code actually _does_
2. Change the code to match the public declaration

I originally went with option (2), but [you asked me to change it to pass in `Delta`s](https://github.com/quilljs/quill/pull/3845/#discussion_r1278289110). So we need to decide which of those two options we want to go with and I'm happy to make the changes, but the point is that _something_ will need to change here as the type annotations are revealing.

Reply:
I added the missing types without changing the actual code.